src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -38,24 +39,30 @@
 FILENAME_LENGTH = 255
 LOG_NAME_REPLACE_STR = "##NAME##"
 DEFAULT_PROCESS_TIMEOUT = 3600
+DEFAULT_SUITE_TIMEOUT = 3600
 DEFAULT_TEST_TIMEOUT = 900
 MAX_LENGHT = 4096 if not constants.IS_WIN else 8191
 
 def parse_arguments():
     parser = ArgumentParser()
     exec_file_path_help = "Path to the test executable file"
     cache_path_help = "Path to the cache file with test_name list sorted by execution time. .lst file!"
-    worker_num_help = "Worker number. Default value is `cpu_count-1` "
+    worker_num_help = "Worker number. Default value is `cpu_count` "
     working_dir_num_help = "Working dir"
     process_timeout_help = "Process timeout in s"
     parallel_help = "Parallel over HW devices. For example run tests over GPU.0, GPU.1 and etc"
+    split_unit_help = "Split by test or suite"
+    repeat_help = "Number of times to repeat failed and interrupted tests"
 
     parser.add_argument("-e", "--exec_file", help=exec_file_path_help, type=str, required=True)
     parser.add_argument("-c", "--cache_path", help=cache_path_help, type=str, required=False, default="")
-    parser.add_argument("-j", "--workers", help=worker_num_help, type=int, required=False, default=(os.cpu_count() - 1) if os.cpu_count() > 2 else 1)
+    parser.add_argument("-j", "--workers", help=worker_num_help, type=int, required=False, default=(os.cpu_count()) if os.cpu_count() > 2 else 1)
```

Comment:
the same. Why do we need the code `if os.cpu_count() > 2 else 1`??

Reply:
removed